### DISCUSSION TITLE
Branch protection bypass not working

### DISCUSSION THREAD

**Discussion by dbeilin** [community/community#113999] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,
I created 2 rulesets:
- A generic protection rule for `main` which applies to everyone, no bypass: [screenshot](https://github.com/community/community/assets/144586547/a1bd8a16-0242-413d-9dad-fdd9f9eb8373)
- A rule that has a requirement for a specific status check to pass, by there's a bypass for my GitHub App: [screenshot](https://github.com/community/community/assets/144586547/967461e9-6a85-4115-b0bd-12e50772fd7d)

I don't understand why, but when the app opens a PR, it still fails to bypass the status check. It still says it's required:
<img width="868" alt="SCR-20240324-lfqp" src="https://github.com/community/community/assets/144586547/7a17b1b7-be75-4444-8239-4536dccb3b55">

I also tested it without a GitHub app, I tried to add myself to a test group I created and set that group in the bypass list, but it didn't work either.
Even when I consolidate those rules into a single rule, it also doesn't work...

What am I missing here?

**Comment by anerdwithaknife** [comment]
@dbeilin I am facing the same issue. Did you figure out what you were missing or did you close the issue because you choose another route?
